### ISSUE TITLE
Full Accessibility & 508 Office Audit | Anchor Tags (Copy deep link) | C& L Team

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#25222]
## Steps to complete the Full Accessibility & 508 Audit: 
- [ ] VFS: Verify all accessibility defects that were identified at Staging Review have been fixed before requesting a full audit *(Platform understands that other teams may be accountable for fixing the defects identified at Staging Review. These defects are exempt and will not block a team from scheduling this touchpoint)*
- [ ] VFS Product Manager: Create this issue and fill in your team and feature name in the title
- [ ] VFS: Link to this issue once created in [#vfs-platform-support](https://dsva.slack.com/archives/CBU0KDSB1) in Slack; tag @ Trevor Pierce
- [x] Platform: Create sub-issues attached to this epic with findings. Audits are performed on a first-come-first-serve basis and are scheduled with each biweekly sprint
- [ ] VFS: Coordinate with the VA 508 Office to address accessibility findings
- [ ] Platform: Update [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=1116695886)
- [ ] VFS: Close this ticket when all steps above are completed.

## Artifacts
- Access information for the tool or feature in staging. Do not put staging credentials in your `va.gov-team` ticket; store or reference them in a .md file in the `va.gov-team-sensitive` repository
     - https://staging.va.gov/coronavirus-veteran-frequently-asked-questions/
- Link to verification that accessibility checks are complete:
  - (preferred) Link to your [TestRail Platform accessibility test plan template](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc).
OR
  - Link to a doc in your product folder that indicates all required checks from the accessibility [staging review prep](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md) doc are complete.

     - https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content-localization/accessibility-checklist-anchor-tags.md

- List of known issues (with links to tickets when applicable)
     - https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/23976

## Additional information
As part of this review, Platform will send your product to VA's 508 office for approval (as applicable).

Please refer to the [Platform Collaboration Cycle README](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/README.md) on Github for more information about the Collaboration Cycle.

**Comment by shiragoodman** [comment]
@joshkimux @AngelaFowler82 FYI

**Comment by joshkimux** [comment]
@SKasimow I've finished my full accessibility audit!

**Comment by SKasimow** [comment]
Thanks @josh! 

CC @jenniferlee-dsva

**Comment by shiragoodman** [comment]
hey @SKasimow  - please be sure to follow the Steps outlined above and coordinate with VA 508 Office to address findings. Thanks!

**Comment by SKasimow** [comment]
@shiragoodman - can you clarify whether we need to have all issues addressed before requesting the full audit or whether we can send through with open tickets for known issues? 

> VFS: Verify all accessibility defects that were identified at Staging Review have been fixed before requesting a full audit

> List of known issues (with links to tickets when applicable)

cc: @youngfreezyVA

**Comment by noahgelman** [comment]
We advise teams to fix the findings before sending to VA 508, but we can advise that you should include these two known issues in your request for an audit. That way VA knows about them and doesn't spend time writing them up as defects a second time.

**Comment by SKasimow** [comment]
@noahgelman Sorry - not following; should I send to VA 508 if the issue has not been fixed yet and include this as a known issue? Or want until a fix goes in??

**Comment by noahgelman** [comment]
@SKasimow 
You can send to VA 508 even if it's not fixed yet, just include as a known issue

**Comment by SKasimow** [comment]
@noahgelman Ah - thank you for clarifying. I will open a ticket on this.

**Comment by SKasimow** [comment]
Closing since we removed links and concluded experiment.
